coramin/utils/coramin_enums.py

```diff
@@ -0,0 +1,12 @@
+from enum import IntEnum
```

Comment:
I think we should use regular Enum here instead of IntEnum. It prevents (RelaxationSide.UNDER == 1) from returning True, forcing one to use the named enumerations (which is better for clarity).